Exercises/data-types-overview/data-types-overview.js

```diff
@@ -0,0 +1,27 @@
+//1. Declare a variable on each type using two options: plain creation, creation with constructor.
+let age = 32;
+let position = 'Product Owner';
+let priorExperience = true; 
+let otherDetails = null; 
+
+class User {
```

Comment:
was meant to create number/string/ etc. using their classes. We haven't talk about that so you can fix it after lesson